src/Targets/Microsoft.FSharp.DesignTime.targets

```diff
@@ -34,9 +34,24 @@
           >
 
     <ItemGroup>
+      <_CompilerCommandLineArgs Include="@(FscCommandLineArgs, ';');"/> 
+    </ItemGroup>
+
+  </Target>
+
+
+<!-- Saved
```

Comment:
That is an artifact of what I was doing, I will remove it before we consider this done.